crates/oxc_span/src/source_type/mod.rs

```diff
@@ -20,13 +20,13 @@ pub use error::UnknownExtension;
 #[serde(rename_all = "camelCase")]
 pub struct SourceType {
     /// JavaScript or TypeScript, default JavaScript
-    pub(super) language: Language,
+    pub language: Language,
```

Comment:
Issue: can we avoid making these public? Barring that, we should make it non_exhaustive